releasenotes/notes/dbm-oracle-2adb01ab985667c2.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Oracle DBM monitoring with activity sampling.
```

Comment:
The CHANGELOG is aimed at users. Could we update this to better explain what feature are added, how to use it or maybe a link to our official doc ?